README.md

```diff
@@ -148,6 +148,7 @@ type Profile = {
   * `logoutUrl`: base address to call with logout requests (default: `entryPoint`)
   * `additionalLogoutParams`: dictionary of additional query params to add to 'logout' requests
   * `logoutCallbackUrl`: The value with which to populate the `Location` attribute in the `SingleLogoutService` elements in the generated service provider metadata.
+  * `singleLogoutServiceBinding`: if set to `HTTP-Redirect`, sets the @Binding attribute in the `SingleLogoutService` element to `urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect`. Default is `urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST`.
```

Comment:
It isn't clear with this setting if it should be set to just the suffix or to the entire value. I think this could be cleared up if the value had to be set in its entirety. Thus, one would set it to `urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect`. not `HTTP-Redirect`. The code to use this value wouldn't do string concatenation.